src/Cms/App.php

```diff
@@ -92,13 +92,17 @@ public function __construct(array $props = [], bool $setInstance = true)
         // register all roots to be able to load stuff afterwards
         $this->bakeRoots($props['roots'] ?? []);
 
-        // stuff from config and additional options
-        $this->optionsFromConfig();
-        $this->optionsFromProps($props['options'] ?? []);
-        $this->optionsFromEnvironment();
-
-        // register the Whoops error handler
-        $this->handleErrors();
+        try {
+            // stuff from config and additional options
+            $this->optionsFromConfig();
+            $this->optionsFromProps($props['options'] ?? []);
+            $this->optionsFromEnvironment();
+        } finally {
+            // register the Whoops error handler
+            // put into try/catch/finally block to to ensure Whoops registered
+            // if there is a problem loading the configurations
```

Comment:
Suggestion:

```
// register the Whoops error handler
// inside of a try-finally block to ensure it's still
// registered even if there is a problem loading the configurations
```

Reply:
I'm afk, you can push the commit if you're available.

Reply:
I can do it tonight